### DISCUSSION TITLE
flameshot gui (hyprland)

### DISCUSSION THREAD

**Discussion by zakissimo** [flameshot-org/flameshot#3410] [Q&A]
On hyprland executing `flameshot gui` returns 1. But clicking on the systray icon works fine. What command should I run to get the same effect ? I want to bind it to print screen. Thanks.

**Comment by jack9603301** [comment]
Did you compile and install it? I also encountered this problem, however I found that when I write the configuration like this, it works fine:

```
exec-once = flameshot
bind = $mainMod, Print, exec, flameshot gui
bind = $mainMod + SHIFT, Print, exec, XDG_CURRENT_DESKTOP=sway flameshot gui -r | wl-copy
```

**Reply by zakissimo** [reply]
Sorry, I had to reboot and now it works fine (i don't really know why)

**Reply by jack9603301** [reply]
I also encountered this problem, but after configuring according to the configuration method I provided above, no problem has been found so far. You can try it.
